core/exception/interrupt_spec.rb

```diff
@@ -0,0 +1,35 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+describe "Interrupt.new" do
```

Comment:
Also those specs in `exception_spec.rb` should probably move to their own files and describe the exceptions in more details in the long term.